src/PelJpeg.php

```diff
@@ -374,9 +374,12 @@ public function setICC(PelJpegContent $icc)
      */
     public function getExif()
     {
-        $exif = $this->getSection(PelJpegMarker::APP1);
-        if ($exif instanceof PelExif) {
-            return $exif;
+        $sections_count = count($this->sections);
+        for ($i = 0; $i < $sections_count; $i ++) {
+            $section = $this->getSection(PelJpegMarker::APP1, $i);
+            if ($section instanceof PelExif) {
+                return $section;
+            }
```

Comment:
This function now returns the first valid Exif section? Please add a note to the method's header.

Reply:
Done

Reply:
thanks